pep-0698.rst

```diff
@@ -321,9 +320,9 @@ As described above, adding ``__override__`` may fail at runtime, in which
 case we will simply return the argument as-is.
 
 In addition, even in cases where it does work it may be difficult for users
-to correctly work with multiple decorators, because getting the ``__override__``
-field to exist on the final output requires understanding the implementation
-of each decorator:
+to correctly work with multiple decorators, because successfully setting
+the ``__override__`` field on the final output requires understanding
```

Comment:
```suggestion
to correctly work with multiple decorators, because successfully ensuring
the ``__override__`` attribute is set on the final output requires understanding
```

* Tweak the wording to more accurately reflect the apparent intent here is ensuring the attribute still exists on the final chained output, as opposed to directly setting it there
* Isn't this an attribute, not a "field" (whatever that is?)

Reply:
@stroxler, what's your opinion on the semantics here?

Reply:
I think @CAM-Gerlach's wording is better (although the initial proposal was better than the original, which was not very clear at all).